### ISSUE TITLE
TOSCA node types fail on Ubuntu VM

### ISSUE THREAD

**Issue by alexmaslenn** [SODALITE-EU/iac-platform-stack#31]
## Current behaviour 

TOSCA node types like:

- sodalite.nodes.ImageBuilderEnv
- sodalite.nodes.RegistryCertificate
- sodalite.nodes.TLS.Certificate

fail on Openstack VM with OS other than centos

## Expected behaviour

Ubuntu installation is executed without errors

**Comment by mihaTrajbaric** [comment]
tl;dr: If targeting non-centos with openstack blueprint, do not forget to `export OPERA_SSH_USER=ubuntu`


# Explanation

## Testing
I preformed 2 tests:
### deploy_local.sh on ubuntu VM
I manually created `ubuntu-20.04 x86_64` VM, cloned this repo and run deploy_local:
```shell script
git clone https://github.com/SODALITE-EU/iac-platform-stack.git
cd iac-platform-stack
./deploy_local.sh deploy
```
It deployed without any issues
### deploy_openstack.sh with ubuntu VM
I cloned this repo on my workmachine and changed these two fields in `input.yaml.tmpl` to ubuntu values:
```yaml
image-name: "ubuntu-20.04 x86_64"
username: ubuntu
```
Then I run deploy_openstack:
```shell script
./deploy_openstack.sh deploy [openrc.sh_file_path]
```

And it failed (node `docker-private-registry-server-certs`, type `sodalite.nodes.RegistryServerCertificate`)
```shell script
Failed to connect to the host via ssh: centos@10.10.43.232: Permission denied (publickey,password).
```

The reason for the failure is xopera by default tries to login with user centos ([line of code](https://github.com/xlab-si/xopera-opera/blob/dded9cd9d7d3cd007ac379caa6bceb3fcd82c56b/src/opera/executor/ansible.py#L22) to blame), but it can be overridden by exporting this env var :
```shell script
export OPERA_SSH_USER=ubuntu
```

When I added export to `deploy_openstack.sh`, it worked flawlessly.

## Should we do anything?
Do you think we should change deploy_local.sh?
Since it targets centos distro, I don't see a need for exporting `OPERA_SSH_USER`...

Should we add a note about this in iac-platform-stack docs?
